master/buildbot/steps/source/p4.py

```diff
@@ -231,12 +232,18 @@ def _buildVCCommand(self, doCommand):
         command.extend(doCommand)
 
         def encodeArg(arg):
-            if isinstance(arg, string_types):
+            if isinstance(arg, binary_type):
```

Comment:
I was more thinking about the first part which is the reverse for ascii2unicode

I was thinking about writing this as
```
if isinstance(arg, tuple):
     return (arg[0], encodeUTF8(arg[1]), arg[2])
else:
     return encodeUTF8(arg)
```